docs/extras/xdebug.md

```diff
@@ -14,6 +14,11 @@ To use XDebug with PHPStorm, change the `php_xdebug_idekey` variable as shown be
 ```yaml
 php_xdebug_idekey: PHPSTORM
 ```
+In order to pass the Web Server Debug Validation, you'll need to also add the following your `config.yml`
+
+```yaml
+php_xdebug_remote_host: "{{ vagrant_hostname }}"
```

Comment:
I'm guessing @thom8 means _inside_ the VM (e.g. the guest); hostsupdater works on the host only.